### ISSUE TITLE
Picture in Picture Gets Stuck Randomly

### ISSUE THREAD

**Issue by breakthetargets** [dekuScrub731/picture-in-picture#14]
Hey there, picture in picture seems to be getting stuck on screen randomly with no way to remove it without having to close RuneLite, even turning off the plugin doesn't seem to work at all as shown by the screenshot here: 
![image](https://user-images.githubusercontent.com/81792782/134390674-6ba512d7-5491-46ec-91f0-4091cbc601f4.png)

**Comment by the-fifth** [comment]
Can reproduce this, if you open the plugin hub while PIP is active it becomes stuck.

**Comment by dekuScrub731** [comment]
@the-fifth - Thanks for the steps to reproduce. Taking a look now.

**Comment by dekuScrub731** [comment]
This change will be reflected once v1.4.1 is merged into the RL plugin hub
